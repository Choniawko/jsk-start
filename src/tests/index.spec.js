import { getCurrMonthArr, getLastDay } from "../index"

describe("calendar", () => {
  it("returns array of days", () => {
    // GIVEN
    const currYear = 2019
    const currMonth = 4
    // WHEN
    const weeks = getCurrMonthArr(currYear, currMonth)
    // THEN
    expect(weeks).toMatchSnapshot()
  })

  it("should return last day of the month", () => {
    // GIVEN
    const currYear = 2019
    const currMonth = 4
    const expectedDay = 31
    // WHEN
    const last = getLastDay(currYear, currMonth)
    // THEN
    expect(last).toEqual(expectedDay)
  })
})
