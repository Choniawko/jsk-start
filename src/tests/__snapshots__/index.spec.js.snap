// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calendar returns array of days 1`] = `
Array [
  Array [
    0,
    0,
    1,
    2,
    3,
    4,
    5,
  ],
  Array [
    6,
    7,
    8,
    9,
    10,
    11,
    12,
  ],
  Array [
    13,
    14,
    15,
    16,
    17,
    18,
    19,
  ],
  Array [
    20,
    21,
    22,
    23,
    24,
    25,
    26,
  ],
  Array [
    27,
    28,
    29,
    30,
    31,
    0,
    0,
  ],
]
`;
